docs/requirements.txt

```diff
@@ -0,0 +1,36 @@
+alabaster==0.7.11
```

Comment:
:(:(:( But I know you tried to find a way not to need this...

Reply:
haha. they import it. Maybe it's unused? sphinx wants it. It wasn't me!!!

Reply:
Oh, I was talking about the whole file. I just don't see a way to comment on the file itself.